### PR TITLE
adding a privacy policy

### PR DESCRIPTION
**Pull request by trentwirth**

### CODE REVIEW

File: docs/privacy_policy.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Privacy Policy
+
+We respect and value your privacy. This Privacy Policy outlines how we collect, use, and protect any personal or anonymous data we may collect from users of our software. By using our software, you agree to the terms of this Privacy Policy.
+
+## Collection of Anonymous User Data
+
+We may collect anonymous user data when you use our software. This information includes a "user ping" of data [posted](https://requests.readthedocs.io/en/latest/user/quickstart/#post-a-multipart-encoded-file) to [pipedream](https://pipedream.com/) every time the software is used. This data helps us understand how we can make our software better users. We may also use this information to demonstrate to agencies and corporations that people are using our software, which may help us to grow the project.
+
+The data we currently collect includes:
+- your IP Address
```
Comment (jonmatthis) [inline docs/privacy_policy.md:10]: explain we can't *not* get an IP because it's an internet based ping

File: docs/privacy_policy.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- The time the "user ping" was sent 
+- information about the hardware of the cameras
```
Comment (jonmatthis) [inline docs/privacy_policy.md:12]: not true - and also too specific

Just say some thing general and link to the `pipedream pings` class in the repo

File: docs/privacy_policy.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- information about the 3D positioning of the cameras 
+
+This list may change over time, but we will always keep it up to date. If you wish to [inspect the code yourself](https://github.com/freemocap/freemocap), you are welcome to, as the software is and always will be free and open source. Alternatively, if you do not wish to share your data, you may turn off "user pings" at any point in time. To turn off user pings, uncheck the "Send User Pings" box on the home screen of the FreeMoCap GUI.
```
Comment (jonmatthis) [inline docs/privacy_policy.md:15]: > This list may change over time, but we will always keep it up to date. 

never say something crazy like this 😂

File: mkdocs.yml [hunk 4]
```diff
@@ -23,6 +23,7 @@ nav:
   - 'how_to_guides/index.md'  # an index of clickable links to various guides
   - 'terminology/terminology.md'
   - 'roadmap/roadmap.md'  # actually the roadmap
+  - 'privacy_policy.md'
```
Comment (jonmatthis) [inline mkdocs.yml:26]: attach a screenshot of how it looks through `mkdocs serve` please?

**Review by jonmatthis — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 4 above)
Comment (trentwirth) [reply mkdocs.yml:26]: image

**Review by jonmatthis — APPROVED** [review]
Looks great!

**Pull request merged**